specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServers/tspconfig.yaml

```diff
@@ -15,10 +15,12 @@ options:
     emit-lro-options: "all"
   "@azure-tools/typespec-csharp":
     flavor: azure
-    emitter-output-dir: "{output-dir}/{service-dir}/{namespace}"
+    package-dir: "Azure.ResourceManager.MySql"
+    package-name: "{package-dir}"
     clear-output-folder: true
     model-namespace: true
-    namespace: "Azure.ResourceManager.MySql"
+    namespace: "{package-dir}.FlexibleServers"
```

Comment:
```yaml
    package-name: "Azure.ResourceManager.MySql"
    namespace: "{package-name}.FlexibleServers"
    emitter-output-dir: "{output-dir}/{service-dir}/{namespace}"
    clear-output-folder: true
    model-namespace: true
```
This would make more sense since `package-dir` is deprecated.

Reply:
@MaryGao Could you check why tsp config validation doesn't report `emitter-output-dir` missing on this?